ovos_local_backend/backend/crud.py

```diff
@@ -0,0 +1,393 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import base64
+
+import ovos_local_backend.database as db
+from flask import request
```

Comment:
this was on purpose as i found it made code more readable since `request` is so similar to the `requests` lib and makes my eyes bleed, both styles were used in the file and i decided to pick this one